lib/rules/lighter-image-react.js

```diff
@@ -0,0 +1,71 @@
+/**
+* @fileoverview Rule to avoid heavy image file formats
+* @author Martyna Babiak
+*/
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const moduleVisitor = require("eslint-module-utils/moduleVisitor");
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+/** @type {import('../shared/types').Rule} */
+module.exports = {
+    meta: {
+        type: "problem",
+
+        docs: {
+            description: "Avoid heavy formats of image files",
+            category: "Code improvements",
+            recommended: false,
+            url: ""
```

Comment:
I think the url needs to be added here - with wrong eslint reference: https://eslint.org/docs/rules/<rule_name>
It will be fixed in next PR